### PR TITLE
Add DFK Chain Mainnet Network

### PR DESCRIPTION
**Pull request by friskyfoxdk**
Changes proposed in this pull request:
- Add DFK Chain Mainnet Network

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -1172,6 +1172,20 @@
     "start": 1715902,
     "logo": "ipfs://QmTogMDLmDgJjDjUKDHDuc2KVTVDbXf8bXJLFiVe8PRxgo"
   },
+  "53935": {
+    "key": "53935",
+    "name": "DFK Chain",
+    "chainId": 53935,
+    "network": "mainnet",
+    "multicall": "0x5b24224dC16508DAD755756639E420817DD4c99E",
```
Comment (ChaituVR) [inline src/networks.json:1180]: Hi @friskyfoxdk Need a way to verify multicall code through explorer, make sure it is verified and published to explorer

https://subnets.avax.network/defi-kingdoms/address/0x5b24224dC16508DAD755756639E420817DD4c99E?transfers=code

File: src/networks.json [hunk 2]
```diff
@@ -1172,6 +1172,20 @@
...
+    "rpc": [
+      "https://subnets.avax.network/defi-kingdoms/dfk-chain/rpc",
+      "https://avax-dfk.gateway.pokt.network/v1/lb/6244818c00b9f0003ad1b619/ext/bc/q2aTwKuyzgs8pynF7UXBZCU7DejbZbZ6EUyHr3JQzYgwNPUPi/rpc"
```
Comment (ChaituVR) [inline src/networks.json:1183]: we can remove this node , at this point we are only looking to add full archive node, but this doesn't look like so

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json (hunk 1 above)
Comment (friskyfoxdk) [reply src/networks.json:1180]: Sorry for the delay in this, it's been an effort to get the validation working for this, but we got it: https://subnets.avax.network/defi-kingdoms/address/0x5b24224dC16508DAD755756639E420817DD4c99E?transfers=code

File: src/networks.json (hunk 2 above)
Comment (friskyfoxdk) [reply src/networks.json]: Removed it

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**